Automattic/wp-calypso
# Domains: Improve domain search availability message for subdomains

## Proposed Changes
We noticed that subdomains were incorrectly treated as root domains in the availability message displayed on the domain search component - the availability result is returned correctly, but subdomains were handled as root domains. This PR fixes it.

## Testing Instructions
- Go to `/domains/add/:site-slug/`;
- Try searching for any subdomain that is already registered;
- Ensure you will see the correct message for subdomains instead of the previous one.

### Before
![image](https://github.com/Automattic/wp-calypso/assets/18705930/971fa10f-a9f5-41b0-ab47-ac1b74ab290f)

### After
![image](https://github.com/Automattic/wp-calypso/assets/18705930/eaa1e6a3-18c6-43e5-b4b0-84e6d1a63e9b)

## Pre-merge Checklist
